src/auth_token.c

```diff
@@ -66,6 +66,7 @@ int requestNewAuthToken(char *newToken, size_t len, int r_count)
 
 	struct token_data data;
 	data.size = 0;
+	newToken[0] = '\0';
```

Comment:
I feel memset would be better for the input argument len

Reply:
OK.